k8s/demo/common/idam/idam.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
   namespace: idam
   annotations:
     flux.weave.works/automated: "true"
-    flux.weave.works/tag.idam-api: glob:demo-*
-    flux.weave.works/tag.idam-web-public: glob:demo-*
-    flux.weave.works/tag.idam-web-admin: glob:demo-*
+    flux.weave.works/tag.idam-api: glob:aat-*
+    flux.weave.works/tag.idam-web-public: glob:aat-*
+    flux.weave.works/tag.idam-web-admin: glob:aat-*
 spec:
```

Comment:
dont you also have to change the images tags under "Values"?
approved anyway

Reply:
yes, fixed. brain has gone demo blind